### ISSUE TITLE
Create Presents example story

### ISSUE THREAD

**Issue by phillipadsmith** [TheTyee/design-article.thetyee.ca#131]
Hi @alexgreen,

Just checking that all the elements for a Presents story are in the library.html now? If so, I'll flow in some real series content into an example story.

Phillip.

**Comment by alexgreen** [comment]
> Just checking that all the elements for a Presents story are in the library.html now?

They are not. But lets quickly revisit exactly what these elements are.

Based on these  3 Presents pages ...
http://thetyee.ca/Presents/2015/05/18/National-Aboriginal-Day-Events/
http://thetyee.ca/Presents/2014/01/20/Komagata-Maru/
http://thetyee.ca/Presents/2014/07/14/Margolese-National-Design-for-Living/
... I'd say that the main difference between the Presents page and the standard article page is that the Presents page has no _article__header_ section (the category + headline + deck + date + author organism that all articles start with). Instead, the Presents page is structured in this order:
- "Tyee Presents" header 
- full-width theme/topic/promo banner 
- tool-bar (aka social tools)
- smaller-than-normal headline
- article__body
- optional half-page ad box that appears _**in parallel with**_ the flow of the article

Is that correct?

Many of these elements (except for the article__body and the tool-bar) are substantially different from any elements in a standard article. So we may need new includes for
- "Tyee Presents" header 
- full-width theme/topic/promo banner (might be possible to do this as an extension of the featured-media organism)
- smaller-than-normal headline
- optional half-page ad box that appears _**in parallel with**_ the flow of the article (this will only show up on viewports > 1400px)

These are all things that still need to be built. Should I move them to the top of my priority list?

**Comment by phillipadsmith** [comment]
> Based on these 3 Presents pages ...

Some confusion here, @alexgreen, the links you've provided are Presents "pages," not Presents **stories** (slight difference). 

Here's a Presents story:
http://thetyee.ca/Presents/2014/09/02/Bing-Thom-Margolese-Prize/

Let me know if that helps. Can say more on Tuesday if necessary.

**Comment by alexgreen** [comment]
> the links you've provided are Presents "pages," not Presents stories (slight difference).

Aha. yes that's very different. 

So does this mean that I _don't_ need to design elements for the the Presents "pages"?
When I go to http://thetyee.ca/Presents/ I see that it is almost entirely populated with events and contests similar to the links I sent. So these are all "pages"? Is there some way that I can easily find the actual Presents "stories"?

It appear that the Present "story" example you sent is almost identical to our standard story. The only differences are:
- the addition of the Tyee presents header graphic
- the absence of a feature photo 

Are these the only things I need to account for?

The absence of a feature photo/media is problematic because half-page ads _only_ appear within the featured media section, so if there's no featured media then there can be no half-page ads. And yet many pages/stories/etc(?) linked from the Presents index page do have half-page ads. 
Is there a way we can ensure that Presents "stories" are always displayed with a featured-media object? Then they could be treated like a standard story.

(However, Presents "pages" are very different from standard stories, and if we _do_ have to account for those then it's likely they'll need their own layout.)

> Can say more on Tuesday if necessary.

What happens on Tuesday? If you want to skype I should be able to call you Tues 9am PDT.

**Comment by phillipadsmith** [comment]
> So does this mean that I don't need to design elements for the the Presents "pages"?

For the moment, yes, nothing to be done on pages (until we're complete with stories/articles and moving on to pages).

>  Is there some way that I can easily find the actual Presents "stories"? 

Not easily, no. You can find many with [search](https://www.google.ca/search?q=presents&domains=thetyee.ca&sitesearch=thetyee.ca&client=pub-2188005021510640&forid=1&ie=UTF-8&oe=UTF-8&hl=en&cof=GALT%3A%23008000%3BGL%3A1%3BDIV%3A%23336699%3BVLC%3A663399%3BAH%3Acenter%3BBGC%3AFFFFFF%3BLBGC%3AFFFFFF%3BALC%3A0000FF%3BLC%3A0000FF%3BT%3A000000%3BGFNT%3A0000FF%3BGIMP%3A0000FF%3BFORID%3A1%3B&sa.x=0&sa.y=0&sa=Go), yet it's not _just_ the stories -- you have to hunt around.

> The only differences are:
> the addition of the Tyee presents header graphic
> the absence of a feature photo

Yes, though I think a feature photo could be included (or an option to include/not include).

> The absence of a feature photo/media is problematic because half-page ads only appear within the featured media section, so if there's no featured media then there can be no half-page ads. 

Presents stories are an ad, so they should not really have another ad. That said, I think we'd be fine to require feature photo/media on _new_ stories (not sure we could backfill).

> Is there a way we can ensure that Presents "stories" are always displayed with a featured-media object? Then they could be treated like a standard story.

Yes, we can just make that a requirement of any "story" or "article" on the site.

> What happens on Tuesday? If you want to skype I should be able to call you Tues 9am PDT.

Just meaning to say, I'll be in the office on Tuesday and was not in on Monday, as you now know! :-)

**Comment by alexgreen** [comment]
This all sounds good. 

> Presents stories are an ad, so they should not really have another ad. That said, I think we'd be fine to require feature photo/media on new stories (not sure we could backfill).

Many of the presents stories feature half-page ads from the sponsor and which are directly related to the story content. So I get the impression that this is part of what Michelle is selling to the sponsor. ...so we should find out whether she needs this placement.

**Comment by phillipadsmith** [comment]
> Many of the presents stories feature half-page ads from the sponsor and which are directly related to the story content. So I get the impression that this is part of what Michelle is selling to the sponsor. ...so we should find out whether she needs this placement.

The ads are actually just running through the ad server, so any relation to the story is likely coincidental. 

I checked in with Michelle, and she'd like to continue running ads on presents stories (not necessarily for the advertiser who is paying for the story). 

So, in sum, these are just another story in every sense, other than having some identification above and below the story to indicate that it's sponsored content, as it does now.

Let me know if you have any questions on this. 

In the meantime, I'll start flowing in some content.

**Comment by alexgreen** [comment]
> In the meantime, I'll start flowing in some content.

Cool, lmk when you've got something flowed. I think the identification can be a custom category label and badge.

**Comment by phillipadsmith** [comment]
> Cool, lmk when you've got something flowed. I think the identification can be a custom category label and badge

Sounds good, as long as those won't be confused for a category and a badge when really what we need to identify is that this is "sponsored content," i.e., paid for by another organization. 

I guess my question is: will following other patterns on the site de-emphasize the uniqueness of this content? Visually, we need to tell readers they're reading something different altogether, and not just a category of content.

**Comment by alexgreen** [comment]
> Sounds good, as long as those won't be confused for a category and a badge when really what we need to identify is that this is "sponsored content," i.e., paid for by another organization.
> 
> I guess my question is: will following other patterns on the site de-emphasize the uniqueness of this content? Visually, we need to tell readers they're reading something different altogether, and not just a category of content.

From my point of view, Tyee Presents really _is_ a story category (a category that denotes sponsored content). That's the way it's always been treated on the Tyee's website. http://thetyee.ca/Presents/2014/09/02/Bing-Thom-Margolese-Prize/ is just like any other Tyee story except it has a Tyee Present section ("category"?) header (see attached) and some explainer fine print at the very bottom of the article.

Are you saying we should differentiate Tyee Presents stories more strongly than the Tyee currently does? This seems like an editorial decision. If they want to be really clear that it's sponsored content then the most obvious thing to do is to label it "Sponsored Content". But history leads to assume that they don't want it to be that clear.

![screen shot 2016-05-18 at 8 55 55 pm](https://cloud.githubusercontent.com/assets/151099/15357658/133314b0-1d3b-11e6-8296-e0771e91327e.png)

**Comment by alexgreen** [comment]
> In the meantime, I'll start flowing in some content.

Where are things at with flowing the content? I've pulled the latest from the dev branch but I'm not seeing any presents.html. Unless I'm missing something I may set one up in a couple hours.

**Comment by phillipadsmith** [comment]
Howdy Alex,

Quick response:

1) You're not missing anything! Per our quick exchange this AM, I'm a day or two behind on article page work because of the fundraiser. Back on it on Thursday. But feel free to press on without me! 

2) No _more_ visual differentiation is necessary for Present in the new design, but we should aim for as much differentiation at least. We know that readers complain when "sponsored content" is not clearly and obviously labeled as such -- so I would propose we strive for as much differentiation as makes sense in the new design. 

Over to you, Sir! :-)

## 

http://phillipadsmith.com/#contact

**Comment by alexgreen** [comment]
@phillipadsmith , It appears that Tyee Presents stories can also be part of a (Tyee Presents?) Series, as evidenced by the following box that appears on http://thetyee.ca/Presents/2014/09/02/Bing-Thom-Margolese-Prize/ . However, this presents story doesn't receive a Series badge, so does that mean it's _not really_ a Series? ...and its Series Box isn't a real Series Box either...

Do we need to have dedicated kind of Tyee Presents Series story (versus just a Tyee Presents story that isn't part of a "Presents Series") with a special Tyee Presents Series Box? Do groups of Tyee Presents stories really need to be linked in series, or is this just an odd anomaly?

It would be great if we could dispense with this and just let every Tyee Presents story stand alone. But if we _do_ need to support a "Tyee Presents Series" story grouping then it would probably be more intuitive if we could just use the regular Tyee Series box and badge (although in that case we'll no longer be able to use a badge to denote Tyee Presents).

What's the word?

**Comment by alexgreen** [comment]
@phillipadsmith , I'm now building this page and flowing content into it. May not finish tonight, but I'll push what I get done.

**Comment by alexgreen** [comment]
I've flowed in all the content except for the questionable Tyee Presents series-box. You can find it in the develop branch at local/series.html

1b6ca7a8e0062a4ecd5ec151b0b74ac9b0d63eeb

**Comment by MrBryan** [comment]
So series concerns aside, I can assume that the manner of differentiating "" will be a trivial addition of an element into the normal article layout like with the series badge?

**Comment by MrBryan** [comment]
manner of differentiating "tyee presents"

**Comment by alexgreen** [comment]
@MrBryan that depends on the answers to my questions above, but hopefully yes.

**Comment by alexgreen** [comment]
> Do we need to have dedicated kind of Tyee Presents Series story (versus just a Tyee Presents story that isn't part of a "Presents Series") with a special Tyee Presents Series Box? Do groups of Tyee Presents stories really need to be linked in series, or is this just an odd anomaly?
> 
> It would be great if we could dispense with this and just let every Tyee Presents story stand alone. But if we do need to support a "Tyee Presents Series" story grouping then it would probably be more intuitive if we could just use the regular Tyee Series box and badge (although in that case we'll no longer be able to use a badge to denote Tyee Presents).

This thread indicates that we do indeed need to support Tyee Presents Series:
https://basecamp.com/1929296/projects/3141205/todos/247564043#comment_421729688
Correct, @phillipadsmith ?

I suggest that we don't use the usual Tyee Series designation badge and Series Box. Instead we can use a specific Tyee Presents Series Box in combination with the Tyee Presents badge. Sound good?

**Comment by phillipadsmith** [comment]
> Correct, @phillipadsmith ?

Correct. As per how the site currently works, we'll need to support a box of related stories in a Presents story if they exist.

**Comment by phillipadsmith** [comment]
> So series concerns aside, I can assume that the manner of differentiating "" will be a trivial addition of an element into the normal article layout like with the series badge?

Not an element in terms of Bricolage, however -- technically, the visual indication should be triggered by the Category of the story, e.g., /Presents/

**Comment by alexgreen** [comment]
completed with bdef968e614549910e1bf2e936a36d2c2684e4fc
See http://thetyee.github.io/design-article.thetyee.ca/presents
and http://thetyee.github.io/design-article.thetyee.ca/presents-series
